### PR TITLE
Bump mobile wallet to 1.1.6

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Review by mvaivre — APPROVED** [review]

**Comment by nop33** [comment]
Let's merge this one after internal testing phase completes and before we go for open testing.

**Comment by nop33** [comment]
Let's merge this after https://github.com/alephium/alephium-frontend/pull/818 is merged

**Comment by nop33** [comment]
@mvaivre I included in this PR the fix for:
- #821 

and I also triggered new internal testing builds from here. I hope you don't mind!

Can I please have a review? 🙏

**Comment by nop33** [comment]
@mvaivre I believe I have fixed all issues including:
- slow import of addresses
- being stuck in splash screen when selecting "No"
- missing addresses data after restore

File: apps/mobile-wallet/src/features/splash-screen/SplashScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+*/
+
+import { Image } from 'expo-image'
+import styled from 'styled-components/native'
+
+import splashSrc from '~/features/splash-screen/splash.png'
+
+const SplashScreen = () => (
+  <SplashScreenStyled>
+    <SplashScreenImage source={splashSrc} style={{ resizeMode: 'center', objectFit: 'contain' }} />
```
Comment (mvaivre) [inline apps/mobile-wallet/src/features/splash-screen/SplashScreen.tsx:26]: I get a warning: `[expo-image]: Prop "resizeMode" is deprecated, use "contentFit" instead`

**Review by mvaivre — COMMENTED** [review]
On little thing and then go!

File: apps/mobile-wallet/src/features/splash-screen/SplashScreen.tsx (hunk 1 above)
Comment (nop33) [reply apps/mobile-wallet/src/features/splash-screen/SplashScreen.tsx]: Fixed!

**Pull request merged**